### ISSUE TITLE
many swingley_aslin_2002 "familiar" stimuli not familiar?

### ISSUE THREAD

**Issue by kachergis** [langcog/peekbank-paper#21]
Some stimuli from Swingley & Aslin 2002 are incorrectly labeled as "familiar" (at least their english_stimulus_label is novel--but maybe stimulus_novelty="familiar" is just solely based on the referred to picture? If so, apologies!)

 Wrongly labeled "familiar" -- all from swingley_aslin_2002 (initial sound modified)

`unfamiliar <- c("gall", "mog", "pity", "raby", "tog", "opple", "yitty", "kier", "cur", "vaby")`

**Comment by mzettersten** [comment]
I think this is intentional but it's a good one to talk through!

They are all mispronunciations of familiar words. Are they familiar? Not really. But are they novel in the same sense as novel object-label associations in word learning studies? Not really either.

So, I'm not sure - I can see arguments in both directions.

**Comment by mzettersten** [comment]
maybe we also want a third category for this kind of thing?

**Comment by mcfrank** [comment]
yes, averaging these in is probably averaging in noise... you could mark
them as "mispronunciation"? or the unhelpful "other"?

**Comment by mzettersten** [comment]
ok, I changed all of the mispronounced words to "novel" and pushed the change to OSF.

This is kind of an interim solution because we only have two `stimulus_novelty `options (novel/ familiar), but it seems like the better of the two options given current analysis goals.
